examples/test_feature_example_group_spec.rb

```diff
@@ -0,0 +1,84 @@
+require 'spec_helper'
+
+describe "test feature example group" do
+  feature "test.feature"
```

Comment:
but i cant help to think this "feature" is very similar to turnip's

while turnip's is the feature looking for the steps modules [ref](https://github.com/jnicklas/turnip#scoped-steps). and rutabga is the other way round, steps look for the feature